DataRepo/models.py

```diff
@@ -38,6 +40,21 @@ class TracerLabeledClass:
     MAX_LABELED_COUNT = 20
 
 
+def atom_count_in_formula(formula, atom):
+    """
+    Return the number of specified atom in the compound.
+    Returns None if atom is not a recognized symbol
+    Returns 0 if the atom is recognized, but not found in the compound
+    """
+    substance = Substance.from_formula(formula)
+    count = None
+    if atom in symbols:
+        # composition returns dict of {atomic_weight: count}
+        # symbols is a tuple of elements in atomic weight order
```

Comment:
NB. Atomic number and atomic weight (usually measured in daltons) are 2 different thing.  What composition stores as a key is atomic number.
```suggestion
        # composition returns dict of {atomic_number: count}
        # symbols is a tuple of elements in atomic number order
```